provisioning/osx/configure/src/index.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+import Bluebird from 'bluebird';
+import Docker from 'dockerode';
+import DockerMachine from 'docker-machine';
+import fs from 'fs';
+import path from 'path';
+import R from 'ramda';
+import yaml from 'js-yaml';
+import yargs from 'yargs';
+
+import args from './args';
+import {
+  create as createPortOverrides,
+  merge as mergeComposeFiles,
+  mergeFn as mergeComposeFilesFn,
+  write as writeComposeFiles,
+  writeFn,
+} from './compose-file';
+import { getServices, getComposeFiles } from './config';
+import {
+  create as createNginxConfig,
+  reload as reloadNginx,
+  write as writeNginxConfig,
+} from './nginx';
+import { assign as assignPorts } from './ports';
+import { findWithPublishedPorts as findPublicServices } from './services';
+
+const dockerEnv = Bluebird.promisify(DockerMachine.env);
```

Comment:
If we're on Node 8 here we can use https://nodejs.org/api/util.html#util_util_promisify_original and drop the `Bluebird` dependency